packages/clay-slider/src/__tests__/index.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClaySlider from '..';
+import React from 'react';
+import {cleanup, render} from '@testing-library/react';
+
+describe('ClaySlider', () => {
+	afterEach(cleanup);
+
+	it('renders', () => {
+		const {container} = render(
+			<ClaySlider onValueChange={() => {}} value={10} />
+		);
+
+		expect(container).toMatchSnapshot();
```

Comment:
I mentioned this in another PR as well. But I think it would be best to be more explicit in our tests rather than just snapshots. We mention this a bit on the frontend guidelines as well, https://github.com/liferay/liferay-frontend-guidelines/blob/master/general/testing/ui_components.md

Reply:
Sure, I will decrease the use of snapshots here.